### ISSUE TITLE
meet or met implies pkg

### ISSUE THREAD

**Issue by Anniepoo** [larsyencken/marelle#19]
Enhancement suggestion - 
What if the existance of a meet or met implied pkg?

meet(foobycalc) :- 
      install_foobycalc_somehow.

doesn't this sort of imply that foobycalc is a pkg?

If you like this idea, I'll send PR.

**Comment by larsyencken** [comment]
Ah, it does! But then if you checked `pkg(foobycalc)`, it would go off and execute the (stateful) meet block, so you wouldn't be able to query for a list of packages without side effects. `marelle list` needs to be able to do that.

**Comment by Anniepoo** [comment]
no, you can use clause/2 to find out about foobycalc without executing meet
